src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/HighRiskBreachLayout.tsx

```diff
@@ -0,0 +1,117 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import styles from "./HighRiskBreachLayout.module.scss";
+import CreditCardIllustration from "../images/high-risk-data-breach-credit-card.svg";
+import Image from "next/image";
+
+type HighRiskBreachLayoutProps = {
+  dataBreach: "creditCard" | "socialSecurityNumbers" | "bankAccount" | "PIN";
+};
+
+export const HighRiskBreachLayout = (props: HighRiskBreachLayoutProps) => {
+  const mockedData = [
+    { breachName: "Nike", date: "5/04/17" },
+    { breachName: "Adidas", date: "6/12/18" },
+    { breachName: "Puma", date: "2/22/19" },
+  ];
+
+  const listOfBreaches = mockedData
+    .map((item) => `${item.breachName} on ${item.date}`)
+    .join(", ");
```

Comment:
not sure if it's better to use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat for l10n-ization?